### DISCUSSION TITLE
Version 2.985

### DISCUSSION THREAD

**Discussion by gzsombor** [SkyTubeTeam/SkyTube#1130] [General]
* Use latest NewPipeExtractor to accommodate recent Youtube website changes.

<hr /><em>This discussion was created from the release <a href='https://github.com/SkyTubeTeam/SkyTube/releases/tag/'>Version 2.985</a>.</em>

**Comment by drogga** [comment]


**Reply by AndSm345** [reply]
Thanks you very much for your program.
Excuse me for my English.
Some videos SkyTube can't download.
For example
https://youtu.be/24QdWZlLXd4
https://youtu.be/otg4GTpSawk
https://youtu.be/m9EIsMHN6vo

NewPipe and LibreTube donwloaded its.
LibreTube work faster than SkyTube.

**Comment by drogga** [comment]
**The "Live" and "Shorts" videos are still not shown in the channels !**
@gzsombor - Please make them show up in the "Videos" tab or create 2 more tabs for them.
(actually maybe even a third tab for "Podcasts", YT has that one too...)

**Comment by drogga** [comment]
@AndSm345 Why did you replied to my comment, instead of creating a new one, like I just did and deleted the first one, but since you replied, it's handle still shows up. Please delete your reply and post it again as a new comment, instead of a respond this time (or just create an issue ticket). Thank You.

**Comment by drogga** [comment]
BTW, there's absolutely no need and it's actually pointless to automatically or manually create a discussion topic for each new release.
